neurodamus/io/sonata_config.py

```diff
@@ -166,6 +166,7 @@ def parsedRun(self):
         # "OutputRoot" and "SpikesFile" will be read from self._sim_conf.output
         # once libsonata resolves the manifest info
         parsed_run["OutputRoot"] = self._sim_conf.output.output_dir
+        parsed_run["TargetFileBase"] = self.circuits.node_sets_path
```

Comment:
I find `TargetFileBase` a bit hard to understand. And maybe we don't even need a "BlueConfig"-like named entry

Reply:
do you prefer something like parsed_run["config_nodeset_file"]? or parsed_run["circuits_node_sets_path"]?